README.rst

```diff
@@ -213,3 +213,27 @@ To cleanup the environment delete the container:
    docker rm -f gitlab-test
    docker rm -f gitlab-runner-test
 
+Releases
+--------
+
+A release workflow is triggered every Monday and automatically creates a new
+release if any commits merged to the main branch contain commit message scopes
+that would result in a semantic version bump.
```

Comment:
What are the "commit message scopes" which trigger a new release?  Is it only ``fix`` and ``feat``?

Reply:
Oops thanks, that doesn't make any sense, it's not really scopes but types. Maybe I can link to https://github.com/semantic-release/semantic-release#how-does-it-work here, it should be:
fix -> patch bump
feat -> minor bump

Then any kind of type that includes `BREAKING CHANGE:` in the commit message -> major.

I'll reword this.

Reply:
Done, added types so I hope this is clearer now.